.github/workflows/comment_pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Comment code metrics report to PR
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  comment-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variable
+        id: plum
+        run: echo "::set-output name=PULL_NUMBER::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@master
```

Comment:
I think you know well but just would like to mention the official document.
> Using the default branch of an action may be convenient, but if someone releases a new major version with a breaking change, your workflow could break.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses